cpp/src/jni/arrow_util.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include <iostream>
+
+#define ABORT_ON_FAILURE(expr)                     \
+  do {                                             \
+    const arrow::Status& status_ = (expr);         \
+    if (!status_.ok()) {                           \
+      std::cerr << status_.message() << std::endl; \
+      abort();                                     \
+    }                                              \
+  } while (0);
```

Comment:
Can below code be used to replace this?
https://github.com/oap-project/gazelle-jni/pull/78/files#diff-34fc9ed6a608110dfaa7497275881060f6e96c2e5252d432c45c69d3ae25e332R84-R88